src/main.js

```diff
@@ -14,39 +14,56 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 
 import Memoization from '@/memoization.js'
 
-const Store = require('electron-store')
-const store = new Store()
-
-const log = require('electron-log');
-log.transports.console.format = '[{level}] {text}';
+import Mods from "./mods.js"
+import Resources from "./resources.js"
 
 library.add([
   faExternalLinkAlt, faChevronUp, faChevronRight, faChevronDown, faChevronLeft, 
   faSearch, faEdit, faSave, faTrash, faTimes, faPlus, faPen, faMusic, faLock, 
   faRedo, faStar, faRandom, faMousePointer, faBookmark, faTerminal, faMapPin
 ])
 
+window.isWebApp = (window.isWebApp || false)
+
 Vue.component('fa-icon', FontAwesomeIcon)
 
 Vue.config.productionTip = false
 
-Vue.use(localData, {
-  store: new localData.Store(store.get('localData'))
-})
+const ipcRenderer = (window.isWebApp ? require('@/../webapp/fakeIpc.js') : require('electron').ipcRenderer)
 
-const {shell, ipcRenderer} = require('electron')
-const {port, appVersion} = ipcRenderer.sendSync('STARTUP_GET_INFO')
+// Must init resources first.
+var shell, store, log, port, appVersion
+if (!window.isWebApp) {
+  var {shell} = require('electron')
 
-const Resources = require("@/resources.js")
-Resources.init({
-  assets_root: `http://127.0.0.1:${port}/`
-})
+  const Store = require('electron-store')
+  store = new Store()
 
-// Must init resources first.
-import Mods from "./mods.js"
+  log = require('electron-log');
+  log.transports.console.format = '[{level}] {text}';
+
+  var {port, appVersion} = ipcRenderer.sendSync('STARTUP_GET_INFO')
+
+  Resources.init({
+    assets_root: `http://127.0.0.1:${port}/`
+  })
+} else {
+  store = require('@/../webapp/localstore.js')
+  log = { scope() { return console; } }
+
+  var {port, appVersion} = ipcRenderer.sendSync('STARTUP_GET_INFO')
+
+  Resources.init({
+    assets_root: window.webAppAssetPackHref
```

Comment:
if webapp, initialize Resources with the known asset pack base URL instead of the localhost server used by background.js